### PR TITLE
Adding air filter to HVACMaintenance

### LINKED ISSUE #49: Add furnace filter size
This may not be relevant outside the ENERGY STAR Home Advisor, but we keep track of the filter size for the homeowner. Three numbers: ___ x____x____

### LINKED ISSUE #50: Add date of last furnace filter change
Add the month/year when the filter was last changed

### PR DESCRIPTION
**Pull request by nmerket**
This adds a new complex element `HVACSystemInfoType/HVACMaintenance/AirFilter`. This will show up in `HeatingSystem`, `CoolingSystem`, and `HeatPump`.

baseelements_hvacmaintenance

Fixes #49
Fixes #50

### CODE REVIEW

**Comment by nmerket** [comment]
Yes, it should be there. I added the appropriate `SystemIdentifier`.

File: schemas/BaseElements.xsd [hunk 1]
```diff
@@ -4244,6 +4244,39 @@ https://hpxml.nrel.gov/wiki/WallTypes</xs:documentation>
...
+									</xs:element>
+									<xs:element name="Thickness">
+										<xs:annotation>
+											<xs:documentation>[in]</xs:documentation>
+										</xs:annotation>
+									</xs:element>
+								</xs:sequence>
+							</xs:complexType>
+						</xs:element>
+						<xs:element minOccurs="0" name="DateLastReplaced" type="xs:date"/>
```
Comment (nmerket) [inline schemas/BaseElements.xsd]: Maybe this should be changed to just require month and year. @RebHudson?

**Comment by RebHudson** [comment]
@nmerket I only want month and year

**Comment by RebHudson** [comment]
Can we also add MERV rating?

**Comment by nmerket** [comment]
@RebHudson, noted #82.

**Pull request closed without merging**